README-PT-BR.md

```diff
@@ -0,0 +1,55 @@
+# Visual Studio Code - Open Source
+
+[![Build Status](https://vscode.visualstudio.com/_apis/public/build/definitions/a4cdce18-a05c-4bb8-9476-5d07e63bfd76/1/badge?branchName=master)](https://aka.ms/vscode-builds)
+[![Feature Requests](https://img.shields.io/github/issues/Microsoft/vscode/feature-request.svg)](https://github.com/Microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc)
+[![Bugs](https://img.shields.io/github/issues/Microsoft/vscode/bug.svg)](https://github.com/Microsoft/vscode/issues?utf8=✓&q=is%3Aissue+is%3Aopen+label%3Abug)
+[![Gitter](https://img.shields.io/badge/chat-on%20gitter-yellow.svg)](https://gitter.im/Microsoft/vscode)
+
+[VS Code](https://code.visualstudio.com) é um novo tipo de ferramenta que combina a simplicidade de
+um editor de código com o que desenvolvedores precisam para o seu ciclo principal de edição-compilação-deburação. VS Code
+fornece suporte abrangente de edição e depuração, um modelo de extensibilidade e integração leve com ferramentas existentes.
+
+VS Code é atualizado mensalmente com novas funções e correções de bugs. Você pode baixá-lo para Windows, macOS e Linux no site oficial do[VS Code](https://code.visualstudio.com/Download). Para ter os últimos lançamentos todos os dias, você pode instalar a [Versão Insiders do VS Code](https://code.visualstudio.com/insiders). Essa compilação é feita a partir da branch master e é atualizada diariamente.
```

Comment:
Missing space before the [VS Code] link

Reply:
Solved too, sorry for this inconvenience.